src/apps/administration/index.md

```diff
@@ -0,0 +1,41 @@
+# Administration Overview
+
+The Administration is the merchant interface of every Shopware store.
+
+It comprises UI for business logic, purchasing logic, data, and capabilities. It outlines all internal processes like product inventory tracking, order processing, and customer record management and gets input from frontend applications.
+
+## Build
+
+The [Administration module](https://developer.shopware.com/docs/concepts/framework/architecture/administration-concept) contains infrastructure and assets defined and used in the Administration user interface.
+
+You can modify the appearance and extend capabilities of the Administration by customizing its components such as [template files](https://developer.shopware.com/docs/guides/plugins/plugins/administration/writing-templates), [mixins](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-mixins), [fields](https://developer.shopware.com/docs/guides/plugins/plugins/administration/using-custom-fields), [filters](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-filter), [modules](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-module), [styling](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-styles), etc.
+
+This can be done via [apps](https://developer.shopware.com/docs/guides/plugins/apps/administration) or [plugins](https://developer.shopware.com/docs/guides/plugins/plugins/administration).
```

Comment:
The things apps and plugins can do in this area currently differ somewhat. I'm fine with leaving it as is and having the linked pages specify this, but splitting the overview might also make sense :thinking:

Reply:
May I know what you meant by "splitting the overview"? 🤔